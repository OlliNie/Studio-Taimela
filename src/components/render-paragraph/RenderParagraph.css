.paragraph{
  text-align:center;
  margin:1rem;
  padding-bottom:1rem;
  @media(min-width:1100px){
    width:800px;
    margin-left:0;
    margin-right:0;
  }
}