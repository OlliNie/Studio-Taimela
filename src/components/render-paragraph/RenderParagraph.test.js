import React from 'react';
import { shallow } from 'enzyme';
import RenderParagraph from './RenderPagragraph';

describe('RenderParagraph component', () => {
  it('renders a paragraph from a string', () => {
    const sampleString = 'render me please';
    const wrapper = shallow(<RenderParagraph content={sampleString} />);
    expect(wrapper).toMatchSnapshot();
  });
});
