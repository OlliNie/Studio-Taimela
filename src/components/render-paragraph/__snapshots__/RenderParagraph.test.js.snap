// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RenderParagraph component renders a paragraph from a string 1`] = `
<p
  className="paragraph"
>
  render me please
</p>
`;
